ecosystem/sep-0031.md

```diff
@@ -7,36 +7,47 @@ Title: Direct Payments
+- A **Sending Anchor**: The business offering outbound payment services, who must have a business relationship with the Receiving Anchor.
+- A **Receiving Anchor**: The business offering inbound payment processing.
+- A **Receiving Client**: The owner of the destination financial account.
 
-Alice in Nigeria wants to send money to Bob in Europe. Alice signs up with NigeriaPay to make this payment to send money directly into Bob’s bank account. Bob doesn’t need to do anything, or know anything about this payment, besides letting Alice know what his bank account information is. Alice only needs to deal with her anchor (NigeriaPay).
+At a high level, the following steps are performed to complete a transaction:
 
-NigeriaPay will utilize its European rail, enabled with EuroPay Anchor service, to move the money to EuroPay in order to deposit it into Bob’s bank account.
+1. The Sending Client sends funds from a financial account to the Sending Anchor
+2. The Sending Anchor sends the associated amount of funds to the Receiving Anchor's Stellar account
+3. The Receiving Anchor sends funds to the Receiving Client's financial account
 
-## Prerequisites
+Typically, the Sending and Receiving Clients reside in different regulatory jurisdictions and therefore a payment between their financial accounts must be facilitated by two business entities, the Sending and Receiving Anchors, who have the necessary licenses in their respective jurisdictions.
 
-* An anchor must define the location of their `DIRECT_PAYMENT_SERVER` in their [`stellar.toml`](sep-0001.md). Anchors will find each other's servers by pulling this TOML file from their home domains.
-* Anchors will create bi-lateral agreements to interoperate with each other.  This differs from other protocols in that there is no concept of 'discoverability'.  Anchors should keep a mapping of their partnerships in different regions to their home domains.
-* Each anchor registers a Stellar key with each counterparty anchor they interact with in order to identify themselves via [SEP-10 Web Authentication](sep-0010.md).
+## Example
 
+Alice in Nigeria wants to send money to Bob in Europe. Alice signs up with NigeriaPay to make this payment to send money directly into Bob’s bank account. Bob doesn’t need to do anything, or know anything about this payment, besides letting Alice know what his bank account information is. Alice only needs to deal with her anchor (NigeriaPay). Alice passes this information and her money to NigeriaPay, and NigeriaPay sends those funds to the EuroPay Anchor service, and Europay deposits those funds into Bob’s bank account.
+
+## Prerequisites
+
+* The Receiving Anchor must define `DIRECT_PAYMENT_SERVER` in their [`stellar.toml`](sep-0001.md).
+* The Sending and Reciving Anchors must create bi-lateral agreements to interoperate with each other. This differs from other protocols in that there is no concept of 'discoverability'.
```

Comment:
💭 If there is no concept of discoverability then the use of the stellar.toml for the API endpoint seems irrelevant. The two parties could establish their endpoints with each other without needing to lookup URLs in this file. The two parties may choose to operate over a VPN or some other mechanism. Or is the sending client an independent piece of software that knows how to talk to the SEP-31 parties and needs it?

Reply:
Maybe saying "there is no concept of 'discoverability'" is too strong. 

Signaling to the ecosystem that you support SEP-31 (i.e. making your service discoverable) is still valuable even if a bi-lateral agreement is necessary before the anchors can transact. So I think it makes sense to remove this sentence.